pkgs/build-support/remove-references-to/remove-references-to.sh

```diff
@@ -26,7 +26,16 @@ for i in "$@"; do
     test ! -L "$i" -a -f "$i" && regions+=("$i")
 done
 
+if [[ "${#regions[@]}" -eq 0 ]] ; then
+    if (( "${NIX_DEBUG:-0}" >= 1 )); then
+        echo "removeReferencesTo: no references found" >&2
+    fi
+    exit 0
+fi
```

Comment:
Aj, I misunderstood this. These are not the references, these the regular files from `$@`. I'll update the message and maybe make it `exit 1`